src/kOS/Screen/TermWindow.cs

```diff
@@ -963,11 +961,20 @@ protected void CheckResizeDrag()
             }
         }
         
-        void ShowCharacterByAscii(char ch, int x, int y, bool reversingScreen, int charWidth, int charHeight, float brightness)
+        void DrawCursorAt(char ch, int x, int y, bool reversingScreen, int charWidth, int charHeight, float brightness)
```

Comment:
I believe this is only still being used to draw the block cursor only.
For drawing the rest of the screen, I already changed it to draw one screen line at a time using GUI.Label(), or so I thought.